storey/flow.py

```diff
@@ -158,6 +168,7 @@ def to(self, outlet):
         if outlet._legal_first_step:
             raise ValueError(f"{outlet.name} can only appear as the first step of a flow")
         self._outlets.append(outlet)
+        outlet._inlets.append(self)
```

Comment:
can you explain what does the _inlets list holds?
i would think its list of steps leading to a specific test, and in that case the outlet._inlets should have "self" appended.

Reply:
If by "test" you mean "step" then yeah :)